packages/core/src/contrib/Constraints.ts

```diff
@@ -303,18 +308,20 @@ export const constrDict = {
       const res = sub(d, o);
       return res;
     } else if (t1 === "Circle" && isRectlike(t2)) {
-      const d = ops.vdist(fns.center(s1), fns.center(s2));
-      const textR = max(s2.w.contents, s2.h.contents);
+      const s2BBox = bbox([t2, s2]);
```

Comment:
Avoid bbox use?

Reply:
This instance is necessary since `Square` does not have a `w` or `h` attribute.